runtime/src/main/java/org/corfudb/runtime/view/LayoutBuilder.java

```diff
@@ -7,6 +7,7 @@
 import java.util.List;
 import java.util.Set;
 
+import java.util.UUID;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.UUID'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)